app/hotels/src/GetVersion.js

```diff
@@ -0,0 +1,5 @@
+// @flow
+
+import hotelPackage from '../package.json';
+
+export const getVersion = () => hotelPackage.version;
```

Comment:
Hm, but you did it here... 🤔

I need to read it again what are you doing there actually... 😄